src/PrestaShopBundle/Resources/views/Admin/Sell/Address/Blocks/form.html.twig

```diff
@@ -0,0 +1,139 @@
+            'label': 'Customer email'|trans({}, 'Admin.Orderscustomers.Feature'),
+          }) }}
+        {% endif %}
+      {% endblock %}
+
+      {{ ps.form_group_row(addressForm.dni, {}, {
+        'label': 'Identification number'|trans({}, 'Admin.Orderscustomers.Feature'),
+        'help' : 'The national ID card number of this person, or a unique tax identification number.'|trans({}, 'Admin.Orderscustomers.Help')
+      }) }}
+
+      {{ ps.form_group_row(addressForm.alias, {}, {
+        'label': 'Address alias'|trans({}, 'Admin.Orderscustomers.Feature'),
+        'help' : 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}'
+      }) }}
+
+      {{ ps.form_group_row(addressForm.first_name, {}, {
+        'label': 'First Name'|trans({}, 'Admin.Global'),
+        'help' : 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' 0-9!<>,;?=+()@#"�{}_$%:'
+      }) }}
+
+      {{ ps.form_group_row(addressForm.last_name, {}, {
+        'label': 'Last Name'|trans({}, 'Admin.Global'),
+        'help' : 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' 0-9!<>,;?=+()@#"�{}_$%:'
+      }) }}
+
+      {{ ps.form_group_row(addressForm.company, {}, {
+        'label': 'Company'|trans({}, 'Admin.Global'),
+        'help' : 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}'
+      }) }}
+
+      {{ ps.form_group_row(addressForm.vat_number, {}, {
+        'label': 'VAT number'|trans({}, 'Admin.Orderscustomers.Feature'),
+      }) }}
+
+      {{ ps.form_group_row(addressForm.address1, {}, {
+        'label': 'Address'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {{ ps.form_group_row(addressForm.address2, {}, {
+        'label': 'Address'|trans({}, 'Admin.Global') ~ ' (2)',
+      }) }}
+
+      {{ ps.form_group_row(addressForm.postcode, {}, {
+        'label': 'Zip/postal code'|trans({}, 'Admin.Global')
+      }) }}
+
+      {{ ps.form_group_row(addressForm.city, {}, {
+        'label': 'City'|trans({}, 'Admin.Global')
+      }) }}
+
+      {{ ps.form_group_row(addressForm.id_country, {'attr': {
+        'data-states-url': path('admin_country_states') }}, {
+        'label': 'Country'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {% if addressForm.id_state.vars.disabled is defined and addressForm.id_state.vars.disabled %}
+        {% set rowAttr = {'style': 'display:none'} %}
+      {% endif %}
+
+      {{ ps.form_group_row(addressForm.id_state, {}, {
+        'label': 'State'|trans({}, 'Admin.Global'),
+        'class': 'js-address-state-select',
+        'row_attr': rowAttr|default({})
+      }) }}
+
+      {{ ps.form_group_row(addressForm.phone, {}, {
+        'label': 'Home phone'|trans({}, 'Admin.Global')
+      }) }}
+
+      {% if addressForm.phone_mobile is defined %}
+        {{ ps.form_group_row(addressForm.phone_mobile, {}, {
+          'label': 'Mobile phone'|trans({}, 'Admin.Global')
```

Comment:
I wonder if we need two different fields. And if we do, how to introduce them so that it does not sound too outdated...
🤔

Reply:
Yah we need those 😞 As for how to call it i can only suggest one of the names from the list of stationary phone synonyms https://www.powerthesaurus.org/stationary_phone/synonyms 😄